### DISCUSSION TITLE
扫描组件识别问题

### DISCUSSION THREAD

**Discussion by TingRuazhang** [dotnetcore/BootstrapBlazor#3653] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[BootstrapBlazor.BarCode]控件扫码时有几率出错，一直显示一堆数字无法正常再进行扫码，需要刷新页面后才能重新扫码
![微信图片_20240611151128](https://github.com/dotnetcore/BootstrapBlazor/assets/68175738/d62faa20-fcb8-4667-8d71-4815c6806f9e)

### Expected Behavior

希望可以正常识别二维码，降低出错概率

### Steps To Reproduce

用bootstrap控件二维码生成工具生成一个长字节的二维码，然后使用发送到手机上，用电脑的摄像头扫码，有很大概率出错

### .NET Version

.net8.0

### Anything else?

PM> dotnet --info
.NET SDK:
 Version:           8.0.300
 Commit:            326f6e68b2
 Workload version:  8.0.300-manifests.6751eab7
 MSBuild version:   17.10.4+10fbfbf2e

**Comment by bb-auto** [comment]
@q1098401179 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by TingRuazhang** [comment]
https://www.blazor.zone/barcode-reader 使用的是官网的示例进行扫码的一样出错

**Comment by ArgoZhang** [comment]
@q1098401179 你这问题怎么复现？怎么找问题？你就给一个结果

**Comment by TingRuazhang** [comment]
> @q1098401179 你这问题怎么复现？怎么找问题？你就给一个结果
提供复现的视频可以嘛

**Comment by ArgoZhang** [comment]
总之你得让开发人员有办法发力啊。你就一个结果不准确，这怎么解决问题啊？没有人反映过这个问题

**Comment by TingRuazhang** [comment]
[

https://github.com/dotnetcore/BootstrapBlazor/assets/68175738/4d70ea05-c7ce-453f-9519-4f3bffca93be

](url)

**Comment by TingRuazhang** [comment]
如视频所示没有任何二维码但是一直显示识别到了二维码。用bootstrap控件二维码生成工具生成一个长字节的二维码，然后使用发送到手机上，用电脑的摄像头扫码，有很大概率复现。

**Reply by ArgoZhang** [reply]
好的。我复现一下

**Reply by ArgoZhang** [reply]
测试步骤 
1. 打开条形码生成网站弄一个条形码
2. 手机打开 [www.blazor.zone/](https://www.blazor.zone/barcode-reader)
3. 尝试移动距离进行扫描
4. 扫描成功后自动关闭摄像头
![IMG_2346](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/6fdc3211-858a-44b8-9285-a6d6ab3b98da)

结论：一切正常

**Comment by densen2014** [comment]
搞半天出错的来源,也就是那个长字节的二维码在哪呢

**Reply by ArgoZhang** [reply]
我随便弄了一个条形码，都是没问题的。二维码不行吧

**Comment by TingRuazhang** [comment]
例如这个二维码，我用手机摄像头扫码是不会触发这个问题的。但是电脑摄像头就会有几率触发。附二维码和测试视频如下
![989ed65b0d554fda55b22b963421f63](https://github.com/dotnetcore/BootstrapBlazor/assets/68175738/4da4f4a5-de7e-40fe-a88f-71ff633f81ce)

https://github.com/dotnetcore/BootstrapBlazor/assets/68175738/f5568006-93c3-4225-a9b5-6ba0cd98302e

**Reply by TingRuazhang** [reply]
上面那串json格式是码里实际的内容，但是后面就扫码出错了，必须刷新页面才能重新识别

**Reply by densen2014** [reply]
```
{"StationNum":null,"LinNum":"LCP2401-0008","CustomerName":"安庆中船柴油机有限公司","TaskContent":"柴油机控制屏(单门主机屏)","PlanEndDate":"2024/6/11 0:00:00","Num":1,"UUID":null}
```

**Comment by densen2014** [comment]
**测试视频**
https://youtu.be/6nqLVkX_AfA

我这边测试没任何问题, @q1098401179 你方便的话换个电脑试试

![image](https://github.com/user-attachments/assets/07fe1e60-1849-4c05-9815-6f93ebbc159c)

![image](https://github.com/user-attachments/assets/afcf80e6-e184-40ba-8cf0-c869234467ea)
